services/consensus_submit_message.proto

```diff
@@ -43,13 +60,25 @@ message ConsensusMessageChunkInfo {
     int32 total = 2;
 
     /**
-     * The sequence number (from 1 to total) of the current chunk in the message.
+     * The sequence number (from 1 to total) of the current chunk
+     * in the message.
      */
     int32 number = 3;
 }
 
 /**
- * UNDOCUMENTED
+ * Submit a message for consensus.
+ * <p>
```

Comment:
Is there some heuristic around when messages should have `<p>` tags or should we always favour blank lines for breaks in messages?

Reply:
Always prefer a blank line.
This line is an error that I will fix.

Reply:
Updated.